app/src/main/java/com/example/myapplication/SecondFragment2.java

```diff
@@ -30,24 +30,10 @@
 public class SecondFragment2 extends Fragment {
 
     private FragmentSecond2Binding binding;
-    public enum Chain{
-        NONE,
-        FAILED,
-        SUCCESS
-
-    }
-    public static Chain chainStatus = Chain.NONE;
-    public enum Harmony{
-        NONE,
-        FAILED,
-        TWO,
-        THREE
-
-    }
-    public static Harmony harmonyStatus = Harmony.NONE;
+    MatchData.Chain chainStatus = MatchData.Chain.NOPE;
+    MatchData.Harmony harmonyStatus = MatchData.Harmony.NOPE;
```

Comment:
These don't need to be instance fields; they can be a local variables inside onItemSelected in their respective widget handlers.